mixins/form.jsx

```diff
@@ -137,7 +137,10 @@ module.exports = {
   },
   submit: function(action, props, callback) {
     props.locale = this.props.locales[0];
-    props.currency = this.state.currency.code;
+    var currency = this.state.currency;
+    if (currency) {
+      props.currency = currency.code;
+    }
```

Comment:
This was just a straight up bug that was blocking signups.

state.currency only exists for donations, singup was throwing a console error.